pkg/agent/apiserver/handlers/ovsflows/handler.go

```diff
@@ -97,7 +97,7 @@ func getTableFlows(aq agentquerier.AgentQuerier, table string) ([]Response, erro
 }
 
 func getPodFlows(aq agentquerier.AgentQuerier, podName, namespace string) ([]Response, error) {
-	interfaces := aq.GetInterfaceStore().GetContainerInterfacesByPod(podName, namespace)
+	interfaces := aq.GetInterfaceStore().GetContainerInterfacesByEntity(podName, namespace)
```

Comment:
I feel no need to change this one, as antrea-agent supports Pods only.

Reply:
Yes Antrea-agent supports Pods only. As mentioned offline, the issue is that the `interfaceStore` interface is being implemented by downstream as well, and in that implementation store knows how to get container interface by ExternalEntity name. So wanted to be compatible here for the interface method declaration.

Reply:
I meant could we keep GetContainerInterfacesByPod() for other code, and only change NetworkPolicyController to use GetInterfacesByEntity? Anyway, only NetworkPolicyController is needed.
We are not clear of the final design yet, so I would limit the scope of the changes.

Reply:
By other code you meant modules like 'ovsflows' and 'traceflow' etc. right? In that case we need to define BOTH `GetContainerInterfacesByPod` and `GetInterfacesByEntity` as interface methods, and in our implementation direct `GetInterfacesByEntity` to `GetContainerInterfacesByPod`? Is this what you have in mind?

Reply:
Yes, that is what I think.